zipkin-lens/src/components/App/App.tsx

```diff
@@ -49,25 +45,23 @@ const App: React.FC = () => {
                 {(config) => (
                   <Provider store={configureStore(config)}>
                     <AlertSnackbar />
-                    <I18nProvider i18n={i18n}>
-                      <BrowserRouter basename={BASE_PATH}>
-                        <Layout>
-                          <Route exact path="/" component={DiscoverPage} />
-                          {config.dependency.enabled && (
-                            <Route
-                              exact
-                              path="/dependency"
-                              component={DependenciesPage}
-                            />
-                          )}
+                    <BrowserRouter basename={import.meta.env.VITE_UI_URL}>
```

Comment:
lets keep the variable names the same, as it is less for people to understand, unless literally `VITE_UI_URL` is the required name. 
```suggestion
                    <BrowserRouter basename={import.meta.env.BASE_PATH}>
```